code/modules/client/preferences.dm

```diff
@@ -17,6 +17,8 @@ GLOBAL_LIST_EMPTY(preferences_datums)
 
 	//Antag preferences
 	var/list/be_special = list() //Special role selection
+	/// Global disable for all antags
+	var/special_disable = FALSE
```

Comment:
There is zero reason for this to be on the preferences datum itself

Reply:
The only stuff that should be in /datum/preferences are stuff too complicated/old to make into a datum preference

Reply:
sooo how should i do it instead

Reply:
With the original PR, this would be a toggle preference datum

With the requirement that I have, this could stay a variable on the preferences datum now but should not be saved at all